### DISCUSSION TITLE
How to use ssl in pyodbc

### DISCUSSION THREAD

**Discussion by prashanttest-elastic** [mkleehammer/pyodbc#1160] [Q&A] [answered]
I want to connect to the MS-SQL server with "TrustServerCertificate": "no," but I received an error: SSL Provider: [error:0A000086:SSL routines::certificate verify failed:unable to get local issuer certificate] (-1) (SQLDriverConnect)')

So I want to ask where I need to set the certificate on the client machine. I used linux OS. It's working with "TrustServerCertificate": "yes"
I just want to know where pyodbc looks for certificates. Also, is there any option to pass a certificate in the connection string?

Currently I am using SQlalchemy but I also checked independently with pyodbc and got same error.

Sample Code which i executed: 
```
import pyodbc

def check_connection():
    database_sa = pyodbc.connect("Connection_String")
    cursor = database_sa.cursor()
    cursor.execute("select 1+1")
    print(cursor.fetchall())
    cursor.close()
    database_sa.commit()

check_connection()
```

**Comment by v-chojas** [comment, marked as answer]
This is unrelated to pyODBC. If you're using the official Microsoft ODBC Driver for SQL Server, see here:
https://learn.microsoft.com/en-us/sql/connect/odbc/linux-mac/connection-string-keywords-and-data-source-names-dsns
Look for the "Default Certificate Trust Store Location" table.
